src/Blazor.Extensions.SignalR/HubConnectionBuilder.cs

```diff
@@ -13,7 +14,7 @@ public class HubConnectionBuilder
         /// This method can only be called once.
         /// </summary>
         /// <returns>Return a <see cref="HubConnection"/>.</returns>
-        public HubConnection Build()
+        public HubConnection Build(IJSRuntime runtime)
```

Comment:
I'm not quite sure we should have parameter on this method. If you look at the official SignalR client, it doesn't...

Reply:
Sure, no problem.  How would you rather we obtain the JS Runtime?

Reply:
It's going to need to get injected one way or another, so not sure we can avoid this unless we want the consumer to inject `HubConnectionBuilder` itself, which might be a reasonable idea if we register it with `.AddTransient`

Reply:
It does seem kind of awkward to have it on `Build()`.  What do you think of making it a constructor parameter of `HubConnectionBuilder` instead?  Or is that what you're suggesting and I'm just dense...  :smirk:

Reply:
Yeah, I think both would work.  I'll defer to @galvesribeiro's preference, but by injecting I meant you could have something like:

```
@inject HubConnectionBuilder HubConnectionBuilder
```

And register that for injection.  But IMO the constructor seems more straightforward. 🤷‍♀️

Reply:
Humm... I think it is quite weird indeed... I think the best would be to `.AddTransient` so we can get the runtime from the DI.

Reply:
Okay, I pushed a couple of commits.  Thank you for your feedback.  Please let me know what you think of the changes.

Reply:
Thank you! I think I've forgotten to revert the .editorconfig. Check the other PRs as well as it is the same things basically. Thank you!

Reply:
Are you referring to .editorconfig.bak ?  I don't think there was a file named just .editorconfig...  I'll circle back to the other PRs once this one is done.  It seemed like it would be easier to implement all the feedback changes in one pass.  Thanks!

Reply:
Sorry, I found .editor config... Restoring it now.